src/storage/src/render/debezium.rs

```diff
@@ -38,8 +38,7 @@ where
     let (oks, errs) = input
         .inner
         .unary(Pipeline, "envelope-debezium", move |_, _| {
-            let mut dedup_state = BTreeMap::new();
-            let envelope = envelope.clone();
+            let mut dedup_state = DebeziumDeduplicationState::new(envelope.clone());
```

Comment:
is this just an extra simplification? why don't we need to partition this by partition id anymore?

Reply:
I'm not sure why we needed the partition in the first place since we can track keys and dbz watermarks regardless of partition. So it's a simplification

Reply:
v nice